### ISSUE TITLE
Add/Improve Touch Gestures

### ISSUE THREAD

**Issue by CarJem** [tagspaces/tagspaces#1729]
TagSpaces is great, and honestly these recent optimizations make it seem more usable... however I must really say that the experience on Touch Screens is really lackluster... and this is especially problematic on Android which usually relies on it.

- The Back Button on Android Does Nothing
- Selecting Multiple Items is Near Impossible without a keyboard
- The File Viewer doesn't take swipe left and right gestures for navigation between items
- Long Press always results in a context menu
- And many more...

The interface is more than friendly for touch, however the lack of actual touch gestures make it really irritating.... 

I might attempt to PR the repo to improve the situation... But needless to say this app is really uncomfortable on a touch screen only device.

**Comment by ceperman** [comment]
I agree. I was about to open an issue on the lack of normal Android back button behavior when I found this. Without this feature using the app is extremely frustrating - and there's no way to exit the app normally.

Desktop behavior is great, but I want/need similar on both platforms.

**Comment by CarJem** [comment]
> I agree. I was about to open an issue on the lack of normal Android back button behavior when I found this. Without this feature using the app is extremely frustrating - and there's no way to exit the app normally.
> 
> Desktop behavior is great, but I want/need similar on both platforms.

And it doesn't help that tag spaces is the only application of its kind... I deeply love TagSpaces... But the lack of touch gestures makes it unusable for me on mobile

**Comment by CarJem** [comment]
I have attempted to make some progress towards this with my PR #1733

**Comment by snugsound** [comment]
This is true on Windows as well: I would expect the back/forward buttons on my mouse to navigate accordingly, as this is pretty much standard behaviour for Windows apps.

**Comment by ceperman** [comment]
> ~I have attempted to make some progress towards this with my PR #1733~
> 
> EDIT: I'm withdrawing my attempt due to having trouble implementing it

I find it odd that it's so difficult to implement what is standard behaviour.
